docs/issues/matrix-programs.md

```
@@ -0,0 +1,51 @@
+# Matrix Programs & Matrix Tests
+
+## Status
+
+Open — Programs exist as Items but loaded program tracking and matrix test dice pools are not
+yet implemented.
+
+## Background
+
+In Shadowrun 4e, matrix tests use the same structure as physical skill tests:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)